### ISSUE TITLE
Fail delete on first try WCAG

### ISSUE THREAD

**Issue by jolevesq** [fgpv-vpgf/contributed-plugins#314]
When we try to set the extent for delete on first try (just create the box before adding something). it fails....

Need to add geometry, open - close tools...
